Code/Engine/RendererCore/Lights/LightComponent.h

```diff
@@ -37,13 +38,24 @@ class EZ_RENDERERCORE_DLL ezLightComponent : public ezRenderComponent
   ezLightComponent();
   ~ezLightComponent();
 
+  void SetUsingColorTemperature(bool bUseColorTemperature);
```

Comment:
Can you add some brief comments to these functions, that explains roughly what this is for (for people unfamiliar with setting light colors in Kelvin).
Can also be a link to a website, like Wikipedia.

Reply:
Fixed